std/random.d

```diff
@@ -1270,6 +1270,19 @@ alias Xorshift    = Xorshift128;                            /// ditto
     }
 }
 
+version(OpenBSD)
+{
+    version = ARC4_RANDOM;
+}
+else version(NetBSD)
+{
+    version = ARC4_RANDOM;
+}
```

Comment:
macOS also has the `arc4random` function. Not sure what's best to use.

[1] https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man3/arc4random.3.html